Assets/Src/Scripts/Animator/HandLandmarksController.cs

```diff
@@ -51,14 +58,45 @@ public class HandLandmarksController : MonoBehaviour {
       for (int i = 0; i < _landmarksNum; i++) {
         _handLandmarks[i] = new GameObject($"HandLandmark{i}");
         _handLandmarks[i].transform.parent = transform;
+        _kalmanFilters[i] = new KalmanFilter(0.125f, 1f);
       }
       _screenRatio = 1.0f * ScreenWidth / ScreenHeight;
+      if (handType == HandType.LeftHand) {
+        _fingerTargets[0] = anim.GetBoneTransform(HumanBodyBones.RightThumbProximal);
```

Comment:
When `handType == HandType.LeftHand`, the bone transform is got from `RightThumbProximal`. If this "mirrored" logic is correct, please also add the comments "The output of HandPose detection is mirrored" here, right before or after the `if` clause.